packages/trader/src/Stores/Modules/MT5/mt5-store.js

```diff
@@ -58,6 +58,12 @@ export default class MT5Store extends BaseStore {
 
     @action.bound
     onMount() {
+        const account_type = sessionStorage.getItem('open_mt5_account_type');
```

Comment:
Could we add a comment here, to explain why are we createMT5Account? :thinking: Or refactoring to a function with a better name, so comments won't be required :D

Reply:
sorry, I got too excited and forgot to do that. pushed now.